### DISCUSSION TITLE
Enhanced communication service

### DISCUSSION THREAD

**Discussion by RocKhalil** [railsadminteam/rails_admin#3508] [Ideas]
Hey guys,
Since we all use `rails_admin` on daily basis ( at least I do :-p ), I was thinking of creating a Discord server to keep the devs connected.

In the server, we can have discussions about enhancements, new features, best practices, etc...

This can help in onboarding devs to rails admin and enhance the gem moving forward.

What do you think about it ?

**Comment by codealchemy** [comment]
Ideally everything that might be those discussions would be discoverable in Github - either in the Wiki or the code itself. I like the idea of finding better ways of collaborating on `rails_admin` (I'm using it on a daily basis as well), maybe we could use the discussions here as a first attempt at that (maybe replace Google Groups as well)?

Something like
- Github Discussions
  - Initial discussions for anything that would otherwise go under Issues or the Wiki
  - Support requests
- Github Issues
  - Bug reports
  - Feature requests
- Github Wiki
  - 'Getting started' documentation
  - `rails_admin` concepts / overview
  - Any more nuanced guidance / gotchas around using the gem
  
Thoughts?

**Reply by RocKhalil** [reply]
Totally makes sense, but I was thinking of a more interactive way to communicate where people would answer you in a couple of minutes / hours.
Usually, opening a ticket on Github to ask something might take days for people to reply ( for example this thread :-p ).

For example, I wanted to ask the community about adding a field `from_rails_admin` to all models, that is set to `true` when submitting a form from `rails_admin` for special condition ( like skipping a validation if the user is admin ), but this would take a couple of days to get a couple of answers about this feature, whereas, if there's a `live` discussion, people would reply faster.. no ?

**Reply by codealchemy** [reply]
Fair - it could lead to quicker turnaround on questions like the one you mentioned, but I wouldn't expect that to be guaranteed (faster responses depend on a lot of things, such as adoption of whatever chat service is used and time zone overlap with active users). It also add _another_ thing to keep tabs on (I don't personally use Discord, other folks may have additional restrictions on work devices that would limit engagement out of an otherwise accepted service like Github during work hours)

So not opposed to it, just a few thoughts / concerns.

cc: @railsadminteam/committers for 👀
